### ISSUE TITLE
After Elastic Search import data in local gives good result but not in server

### ISSUE THREAD

**Issue by ravibadoni** [matthaywardwebdesign/aus-search#10]
Hi,
I am getting issue.
I followed all process mentioned in readme. It's giving good results.
But when I setup all in droplet, It do not gives same result I get in my local system.

What can be the issue Can you help?
O is there a way that I can copy elastic search docker volume to my droplet directly?

**Comment by matthaywardwebdesign** [comment]
Hey @ravibadoni 

What result are you getting in your droplet?

**Comment by ravibadoni** [comment]
Not getting correct response in droplet.
For example I searched for
603A, Street Name

In local it gives me correct result with with 1st item with 603A ....

But in droplet I get different results.

How I setup in droplet:
1- I setup in local first. Then created a dump of whole Mongo database and uploaded it to droplet by using mongorestore command.
2- Next I directly run command 

node elasticsearch-create.js
node elasticsearch-import.js

**Comment by matthaywardwebdesign** [comment]
Are you getting completely incorrect results or no results? It is possible that the elasticsearch import failed and as a result not all of the documents were imported.

Are you able to check the total number of documents in the Elasticsearch index on the server compared to your local instance. Also check your disk space as Elasticsearch will go read only if you run out of disk space.

**Comment by ravibadoni** [comment]
I am getting incorrect result.
What I want is that the result I get in local should be the same result I should get form droplet server URL.

I follow the same process I did for local.

1- 32GB left in my droplet, I restarted  node elasticsearch-import.js
2-
In my local
![image](https://user-images.githubusercontent.com/34301405/78862862-ed48c700-7a55-11ea-9ed6-84978cfe39eb.png)



In server I rerun the command for now node elasticsearch-import.js. I deleted image and container of elasticsearch and rerun it

But for now in server I am getting this
![image](https://user-images.githubusercontent.com/34301405/78862888-fd60a680-7a55-11ea-85cd-791d8ad92e15.png)
